extensions/bulk/index.md

```diff
@@ -10,9 +10,15 @@ extension](/extensions/#official-extensions) of the JSON API specification.
 It provides support for performing multiple operations in a request,
 including adding and removing multiple resources.
 
-Servers **SHOULD** indicate support for the JSON API media type's Bulk
-extension by including the header `Content-Type: application/vnd.api+json;
-ext=bulk` in every response.
+Servers **MUST** indicate support for the Bulk extension by including the
```

Comment:
I would prefer wording like "If a server supports the Bulk extension, it MUST..."